### PR TITLE
Rating slider toast

### PR DESCRIPTION
**Pull request by Nishant-96**
Added Home Page (Installation ) , Colors and typography page.
Along with this it includes reset.css and import.css.
Main 1

typography 1
colors 1

### CODE REVIEW

File: Components/Main.css [hunk 1]
```diff
@@ -0,0 +1,283 @@
...
+
+body { 
+    margin: 0;
+    background-color:var(--mainColor);
+  }
+  
+  .header {
+      display: flex;
+      align-items: center;
+      padding: 0 15px;
```
Comment (znjs) [inline Components/Main.css:11]: if possible use relative units

File: Components/Main.css [hunk 2]
```diff
@@ -0,0 +1,283 @@
...
+  
+    .navigation-header{
+        display:flex;
+        margin-left: auto;
+        flex-wrap: wrap;
+    }
+  
+  .navigation-menu{
+      display: inline-block;
+      padding: 0 10px;
```
Comment (znjs) [inline Components/Main.css:44]: if possible use relative units

File: Components/Main.css [hunk 3]
```diff
@@ -0,0 +1,283 @@
...
+  
+  .container-box{
+    margin:1rem;
+    width: 85%;
+    background-color:var(--mainColor);
+    box-shadow: 0px 0px 0.2rem var(--lightBlue);
+  }
+  
+  .container-component{
+    padding:20px;
```
Comment (znjs) [inline Components/Main.css:134]: if possible use relative units

File: Components/Main.css [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+      margin: unset;
+      justify-content: center;
+    }
+  
+    .header{
+      position: absolute;
+    }
+  
+    .content-box{
+      margin-top: 14rem;
```
Comment (znjs) [inline Components/Main.css:257]: if possible use relative units

**Review by znjs — APPROVED** [review]

**Review by Launchpad5682 — APPROVED** [review]

**Review by znjs — APPROVED** [review]

**Pull request merged**